package.json

```diff
@@ -38,6 +38,7 @@
     "workbox-webpack-plugin": "^6.3.0"
   },
   "dependencies": {
-    "bootstrap": "^5.1.3"
+    "bootstrap": "^5.1.3",
+    "html-loader": "0.5.5"
```

Comment:
Please divide commits and make a distinct PR for installing `html-loader`. Also, it seems that your page doesn't use it yet anyways.